tests/resources/Harbor-Pages/Project_Elements.robot

```diff
@@ -20,8 +20,8 @@ ${create_project_button_css}  .btn
 ${project_name_xpath}  //*[@id="create_project_name"]
 ${project_public_xpath}  //input[@name='public']/..//label
 ${project_save_css}  html body.no-scrolling harbor-app harbor-shell clr-main-container.main-container div.content-container div.content-area.content-area-override project div.row div.col-lg-12.col-md-12.col-sm-12.col-xs-12 div.row.flex-items-xs-between div.option-left create-project clr-modal div.modal div.modal-dialog div.modal-content div.modal-footer button.btn.btn-primary
-${log_xpath}  //clr-main-container/div/clr-vertical-nav/div/a[contains(.,'Logs')]
-${projects_xpath}  //clr-main-container/div/clr-vertical-nav/div/a[contains(.,'Projects')]
+${log_xpath}  //clr-main-container/div/clr-vertical-nav/div/div/a[contains(.,'Logs')]
```

Comment:
should be add just "//" will be ok. 
//clr-main-container/div/clr-vertical-nav//a[contains(.,'Logs')]

Reply:
OK,I have changed it